sdk/communication/communication-common/src/identifierModelSerializer.ts

```diff
@@ -167,6 +202,10 @@ const getKind = (serializedIdentifier: SerializedCommunicationIdentifier): strin
     return "microsoftTeamsUser";
   }
 
+  if (serializedIdentifier.microsoftBot) {
+    return "microsoftTeamsBot";
```

Comment:
should it be changed to `microsoftBot`? 🤔 (same for 133th line)

Reply:
@AikoBB thanks for noticing this. I updated